modules/about-developer-perspective.adoc

```diff
@@ -1,12 +1,13 @@
-:_content-type: ASSEMBLY
-[id="odc-about-developer-perspective"]
+// Module included in the following assemblies:
+//
+// web_console/web-console-overview.adoc
+
+:_content-type: CONCEPT
+[id="about-developer-perspective_{context}"]
 = About the Developer perspective in the web console
-include::_attributes/common-attributes.adoc[]
-:context: odc-about-developer-perspective
 
-toc::[]
+The {product-title} web console provides two perspectives; the *Administrator* perspective and the *Developer* perspective. The *Developer* perspective offers several built-in ways to deploy applications, services, and databases. You can view real-time virtualization of rolling and recreating rollouts on the component. You can also view the application status, resource utilization, project event streaming, and quota consumption. You can also share your project with others. You can also troubleshoot problems with your applications by running Prometheus Query Language (PromQL) queries on your project and examining the metrics visualized on a plot. The metrics provide information about the state of a cluster and any user-defined workloads that you are monitoring.
```

Comment:
Similar comment to the one above about the admin perspective. Maybe we could break it up a bit?